packages/gatsby-theme-newrelic/src/components/SEO.js

```diff
@@ -94,6 +94,20 @@ const SEO = ({ title, location, type, children }) => {
           />
         );
       })}
+
+      {/* Android */}
+      <meta name="theme-color" content="--primary-background-color" />
+      {/* Windows */}
+      <meta
+        name="msapplication-navbutton-color"
+        content="--primary-background-color"
+      />
+      {/* iOS Safari */}
+      <meta
+        name="apple-mobile-web-app-status-bar-style"
+        content="--primary-background-color"
+      />
```

Comment:
I was digging into these meta tags and it looks like we only need this one if the site can be installed as a progressive web app. Safari can key off of the `theme-color` tag for the notch color. [Docs here](https://developer.apple.com/library/archive/documentation/AppleApplications/Reference/SafariWebContent/ConfiguringWebApplications/ConfiguringWebApplications.html)